platform/features/timeline/src/capabilities/GraphCapability.js

```diff
@@ -60,7 +60,7 @@ define(
                         result.power,
                         0,
                         domainObject.getModel().capacity, // Watts
-                        domainObject.getModel().capacity,
+                        domainObject.getModel().startingSOC,
```

Comment:
Since Starting SOC is an optional field in the Create form, we should have a fallback when it's been unspecified. My inclination would be to use the capacity as the Starting SOC whenever `startingSOC` is undefined. (This will also have the nice side effect of not requiring any migration of existing timelines which do not have this property defined.)